pubsub/subscriptions/async_pull.go

```diff
@@ -24,8 +24,13 @@ import (
 	"cloud.google.com/go/pubsub"
 )
 
-func pullMsgs(client *pubsub.Client, subName string, topic *pubsub.Topic) error {
+func pullMsgs(projectID, subName string, topic *pubsub.Topic) error {
+	// subName := projectID + "-example-sub"
```

Comment:
It's difficult to give a sample value for `topic` -- maybe we should initialize the topic in the function for all of the samples that take a topic? What do you think?

Reply:
The quickstart includes creating a new topic, can we assume the user already created a topic to subscribe to/pull from if they're trying to use these functions?

Reply:
Added a link to documentation